### PR TITLE
Change default color in sprite editor

### PR DESCRIPTION
**Pull request by jwunderl**
fixes https://github.com/Microsoft/pxt-arcade/issues/863

ezgif com-resize (8)

### CODE REVIEW

File: pxtlib/sprite-editor/spriteEditor.ts [hunk 1]
```diff
@@ -129,7 +129,7 @@ namespace pxtsprite {
             });
 
             this.sidebar = new SideBar(['url("#alpha-background")'].concat(this.colors), this, this.group);
-            this.sidebar.setColor(1);
+            this.sidebar.setColor(3);
```
Comment (pelikhan) [inline pxtlib/sprite-editor/spriteEditor.ts]: what happens if the sprite editor has less than 3 colors?

Comment (jwunderl) [reply pxtlib/sprite-editor/spriteEditor.ts]: Good call, fixed

File: pxtlib/sprite-editor/spriteEditor.ts [hunk 2]
```diff
@@ -129,7 +129,7 @@ namespace pxtsprite {
             });
 
             this.sidebar = new SideBar(['url("#alpha-background")'].concat(this.colors), this, this.group);
-            this.sidebar.setColor(1);
+            this.sidebar.setColor(this.colors.length >= 3 ? 3 : 1);
```
Comment (riknoll) [inline pxtlib/sprite-editor/spriteEditor.ts]: > 3 not >=

Comment (riknoll) [reply pxtlib/sprite-editor/spriteEditor.ts]: Ah, never mind

**Pull request merged**